docs/source/gbd2017_models/risks/low_birthweight_short_gestation/index.rst

```diff
@@ -51,3 +49,169 @@ described below, and relative to the joint TMREL.
+with the lowest risk was identified. This bin differed within each country dataset. To identify the
+universal 500 g and 2-week gestational age category that would serve as the universal TMREL, all bins that were identified as the TMREL was chosen. This is cat55 (40-42ga, 3500-400g) and cat56 (40-42ga, 4000-4500g)
+
+.. note::
+   the TMREL categories listed in GBD 2017 risk appendix is wrong.  
+
+Causes that are affected by LBWSG
+---------------------------------
+
++----------+---------------------------------------------------------+
+| Cause id | Cause                                                   | 
++==========+=========================================================+
+|  302     | diarrheal diseases                                      |
++----------+---------------------------------------------------------+
+|  322     | lower respiratory tract infections                      |
++----------+---------------------------------------------------------+
+|  328     | upper respiratory tract infections                      | 
++----------+---------------------------------------------------------+
+|  329     | otitis media                                            |
++----------+---------------------------------------------------------+
+|  333     | pneumococcal meningitis                                 |
++----------+---------------------------------------------------------+
+|  334     | H influenzae type B meningitis                          |
++----------+---------------------------------------------------------+
+|  335     | meningococcal meningitis                                |
++----------+---------------------------------------------------------+
+|  336     | other meningitis                                        |
++----------+---------------------------------------------------------+
+|  337     | encephalitis                                            |
++----------+---------------------------------------------------------+
+|  381     | neonatal preterm birth complications                    |
++----------+---------------------------------------------------------+
+|  382     | neonatal encephalopathy due to birth asphyxia and trauma|
++----------+---------------------------------------------------------+
+|  383     | Neonatal sepsis and other neonatal infections           |
++----------+---------------------------------------------------------+
+|  384     | hemolytic disease and other neonatal jaundice           |
++----------+---------------------------------------------------------+
+|  385     | other neonatal disorders                                |
++----------+---------------------------------------------------------+
+|  686     | sudden infant death syndrome                            |
++----------+---------------------------------------------------------+
+
+Restrictions
+++++++++++++
+
+Assumptions and limitations
++++++++++++++++++++++++++++
```

Comment:
It might be helpful to separate out a heading called "Vivarium Modeling Strategy" where we list the equations we use and then put the assumptions and limitations section as a subsection within this section. That way we can easily see what GBD does and then what we do for vivarium.

Reply:
Yes, the topmost sections should be something like:
1. General overview of the risk
2. GBD 2017 Modeling Strategy
3. Vivarium Modeling Strategy

And then every other section should be a subsection of one of these three as appropriate.

See the risk model template and cause model template:

https://vivarium-research.readthedocs.io/en/latest/gbd2017_models/risks/risk_exposure_template.html
https://vivarium-research.readthedocs.io/en/latest/gbd2017_models/causes/cause_model_template.html